common/service/pldm/pldm_firmware_update.c

```diff
@@ -115,6 +121,11 @@ uint8_t pldm_vr_update(void *fw_update_param)
 		}
 	}
 
+	if (!hex_buff) {
+		LOG_ERR("First package(offset=0) has missed, so hex_buff get NULL");
```

Comment:
Can you clean up this error message.  Is it indicating the offset for the first package had some missing?  The if statement already is clear that hex_buff is NULL at this point so no need to put that in the error message.